lib/score-manager.js

```diff
@@ -1,48 +1,98 @@
-/** Represents a collection of scoring methods. */
+/** Apply each plugin to the file, task pair and return scores. */
+function applyPlugins (plugins, file, task) {
+  let scores = {}
+
+  Object.keys(plugins).forEach((key, idx) => {
+    scores[key] = plugins[key](file, task)
+  })
+
+  return scores
+}
+
+/** Returns a copy of the given object. */
+function cloneObject (obj) {
+  return JSON.parse(JSON.stringify(obj))
```

Comment:
Hier müssen wir nur vorsichtig sein. Wenn die Tasks nur timestamps haben, dann ist das okay. Wenn wir mit `Date` Objekten arbeiten, dann geht das nicht mehr. 
`JSON.stringify(new Date())` ergibt einen string und den kann `JSON.parse()` nicht wieder in ein `Date` umwandeln.
Sollte aber erstmal ausreichen.